### ISSUE TITLE
Invalid package version deduction

### ISSUE THREAD

**Issue by gander** [lisonge/vite-plugin-monkey#29]
I am adding a package that is in the beta:
```shell
npm install @formkit/vue
npm show @formkit/vue

# @formkit/vue@1.0.0-beta.11
```

Then I set up the CDN:
```ts
'@formkit/vue': cdn.jsdelivr('FormKit', 'dist/formkit-vue.js'),
```

During the build, the following message appears:
```bash
[plugin-monkey] not found module @formkit/vue version, use @formkit/vue@latest
```

The resulting file has an invalid URL to the package:
```js
// @require      https://cdn.jsdelivr.net/npm/@formkit@latest/dist/formkit-vue.js
```

I am using the following workaround:
```ts
'@formkit/vue': [
    'FormKit',
    (version: string) => `https://cdn.jsdelivr.net/npm/@formkit/vue@${version}/dist/formkit-vue.js`,
],
```
```js
// @require      https://cdn.jsdelivr.net/npm/@formkit/vue@latest/dist/formkit-vue.js
```

**Comment by lisonge** [comment]
you can manually fix it by `import json`, I will fix this later

```ts
// vite.config.ts
import pkg from '@formkit/vue/package.json';

/*
'@formkit/vue': [
    'FormKit',
    (version: string) => `https://cdn.jsdelivr.net/npm/@formkit/vue@${pkg.version}/dist/formkit-vue.js`,
],
/*
```

**Comment by gander** [comment]
Not working

```
TS2732: Cannot find module '@formkit/vue/package.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
```
Even with config (crash during building):
```json
"resolveJsonModule": true,
```

**Comment by lisonge** [comment]
`"vite": "3.1.6"`

**Comment by lisonge** [comment]
fixed by [v2.7.1](https://github.com/lisonge/vite-plugin-monkey/blob/v2.7.1/packages/vite-plugin-monkey/CHANGELOG.md)

about `@formkit/vue` bug, see [@formkit/vue#404](https://github.com/formkit/formkit/issues/404)

**Comment by lisonge** [comment]
test example see [playground/resolve-package-json](https://github.com/lisonge/vite-plugin-monkey/tree/v2.7.1/playground/resolve-package-json)
